toolset/benchmark/test_types/plaintext_type.py

```diff
@@ -21,11 +21,11 @@ def verify(self, base_url):
     body = body.lower()
 
     if "hello, world!" not in body:
-      return [('fail', "Could not find 'Hello, World!' in response", url)]
+      return [('fail', """Could not find 'Hello, World!' in response.""", url)]
 
     if len("hello, world!") < len(body):
       return [('warn', """Server is returning %s more bytes than are required.
-        This may negatively affect benchmark performance""" % (len(body) - len("hello, world!")), url)]
+This may negatively affect benchmark performance.""" % (len(body) - len("hello, world!")), url)]
```

Comment:
Yes, this works. The `"""` means to treat tabs/spaces as additional indentation. When I removed them all (as this line does), it lines up with the indentation of the output from the line before. If you look at my comment, it lines up much nicer.